src/predictor.cc

```diff
@@ -9,11 +9,29 @@
 #include <rime/segmentation.h>
 #include <rime/service.h>
 #include <rime/translation.h>
+#include <rime/schema.h>
 
 namespace rime {
 
 Predictor::Predictor(const Ticket& ticket, PredictDb* db)
-    : Processor(ticket), db_(db) {
+    : Processor(ticket),
+      db_(db),
+      max_iteration_(0),
+      max_candidates_(0),
+      page_size_(5) {
+  // load max_iteration_ and max_candidates_
+  auto* schema = ticket.schema;
```

Comment:
It's better to move this logic to `Create` so there's no need to check schema and get config twice.

Reply:
done